SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -1172,11 +1163,33 @@ void USpatialActorChannel::UpdateSpatialPosition()
 			return;
 		}
 	}
+	
```

Comment:
This might be better turned into a function that returns a bool for if the actor matches the update requirements.